guides/keras_nlp/getting_started.py

```diff
@@ -128,6 +128,12 @@
 # Format is (review text tensor, label tensor)
 print(imdb_train.unbatch().take(1).get_single_element())
 
+# Traverse the data manually
+for data_elem in imdb_train:
```

Comment:
I'm not sure we need this, we are printing the exact same thing basically right above right?

Reply:
I added this because it makes it easier for beginners to understand the code, I will remove it if it does not sound good.